components/private/Math/BUILD

```diff
@@ -12,10 +12,38 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("//:material_components_ios.bzl", "mdc_public_objc_library")
+load(
+    "//:material_components_ios.bzl",
+    "mdc_objc_library",
+    "mdc_public_objc_library",
+    "mdc_unit_test_suite",
+)
 
 licenses(["notice"])  # Apache 2.0
 
 mdc_public_objc_library(
     name = "Math",
 )
+
+mdc_objc_library(
+    name = "unit_test_sources",
+    srcs = native.glob([
+        "tests/unit/*.m",
+        "tests/unit/*.h",
+    ]),
+    sdk_frameworks = [
+        "CoreGraphics",
+        "XCTest",
+    ],
+    visibility = ["//visibility:private"],
```

Comment:
Nit: Private is the default visibility and so it's not necessary to provide it explicitly.

https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes

Reply:
Good to know! Addressed in https://github.com/material-components/material-components-ios/pull/6282/commits/8c495c753a2f2316889306d0476c84abb6db630f.